airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -255,7 +251,8 @@ def get_mapped_task_instances(
     else:
         query = query.order_by(TI.map_index.asc())
 
-    task_instances = query.offset(offset).limit(limit).all()
+    task_instances = session.execute(query.offset(offset).limit(limit)).all()
```

Comment:
This query is giving me a tough time. Tests are failing and I can't figure out what's wrong. Will appreciate some help cc @jedcunningham @uranusjr @dstandish

Reply:
Looks like the `total_entries` line isn’t working as expected, values that should be excluded by filters applied above are still counted. The actual result is correct. I’ll try digging deeper…

Reply:
Oh wait this is **not** the part that’s failing! The failing tests actually point to `get_mapped_task_instance_endpoint` above. (This is `get_task_instances`.)

Reply:
I got this. Should work now.

Reply:
Ok. Thanks!